modules/test/playwright/tests/osb-faro-web/page.spec.ts

```diff
@@ -1068,42 +943,32 @@ test(
 	},
 
 	async ({apiHelpers, page}) => {
-		const pageTitle1 = 'My Page 1';
-		const sitePage1 = await createSitePage({
-			apiHelpers,
-			pageTitle: pageTitle1,
-		});
-		const pageTitle2 = 'My Page 2';
-		const sitePage2 = await createSitePage({
-			apiHelpers,
-			pageTitle: pageTitle2,
-		});
+		const pageName = 'My Page 2';
```

Comment:
@rafaellajordao, `pageName` can we rename this to match to `pageTitleXXXX` pattern?